### PR TITLE
Add people to the Prioritization WG

### PR DESCRIPTION
**Pull request by LeSeulArtichaut**
Welcome @mstallmo @Stupremee @Camelid @jechasteen @frxstrem @bawerd @JamesPatrickGill @djcarpe @am-1t @hameerabbasi  :tada:

cc @spastorino
Depends on #431 and #432 for user files.

### CODE REVIEW

**Comment by spastorino** [comment]
:+1: but CI is failing, it seems that `JamesPatrickGill` and `Stupremee` are not on the people directory yet.

**Comment by Stupremee** [comment]
Yea because I add my user in a different PR, to also join the cleanup notification crew

**Comment by LeSeulArtichaut** [comment]
> Depends on #431 and #432 for user files.

:)

**Comment by LeSeulArtichaut** [comment]
I think this is now ready for merging, modulo blocking PRs #431 and #432.
@spastorino Please check my last commit, I added 3 more people including @hameerabbasi who [asked to join](https://rust-lang.zulipchat.com/#narrow/stream/227806-t-compiler.2Fwg-prioritization/topic/New.20members/near/210607722) this morning (👋)

Could someone from the Core team merge #431 and #432 (adds people to Cleanup Crew)? Maybe cc @pietroalbini

File: people/djcarpe.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = 'DJ Carpenter'
+github = 'djcarpe'
+github-id = 59489977
+zulip-id = 59489977
```
Comment (camelid) [inline people/djcarpe.toml:4]: Is this person's GitHub ID exactly the same as their Zulip ID?

Comment (LeSeulArtichaut) [reply people/djcarpe.toml:4]: Yeah, that’s definitely a copy-paste mistake. Good catch!

**Review by spastorino — APPROVED** [review]

**Comment by Stupremee** [comment]
You can rebase now. Both of our PRs got merged 👍

**Comment by LeSeulArtichaut** [comment]
Rebased, CI should pass now

**Review by pietroalbini — APPROVED** [review]

**Pull request merged**